components/filter/filter.js

```diff
@@ -205,6 +206,7 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 				@d2l-hierarchical-view-hide-start="${this._handleDimensionHideStart}"
 				@d2l-hierarchical-view-show-complete="${this._handleDimensionShowComplete}"
 				@d2l-hierarchical-view-show-start="${this._handleDimensionShowStart}"
+				@keyup="${this._handleDimensionHideKeyPress}"
```

Comment:
The `hierarchical-view` was handling this before for `esc`, but we need to add the listener here for the left arrow key.  The` hierarchical-view` will still handles `esc` and `_handleDimensionHideKeyPress` won't be called in that case.